libqtile/widget/launchbar.py

```diff
@@ -210,10 +213,11 @@ def draw(self):
                 # display an icon
                 self.drawer.ctx.set_source(self.surfaces[name])
                 self.drawer.ctx.paint()
-            self.drawer.draw(self.offset + xoffset, icon_width + self.padding)
+            self.drawer.draw(offsetx=self.offset + xoffset,
+                             width=icon_width + self.padding)
             xoffset += icon_width + self.padding
 
-    def calculate_width(self):
+    def calculate_length(self):
```

Comment:
The only alternative I see is adding a `calculate_height()` method and similar for the other s/width/length, and use the right one depending on the orientation of the bar, but IMHO this would complicate the code a lot, also adding many tests, only for the sake of avoiding the word "length". If you confirm you accept the new terms ("length" and "size") I'll expand the proper comments to better explain what they represent, also comparing them to the previous "width" and "height".